q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,219 @@
+Team.
+
+### Powers
+
+The Steering Council has broad authority to make decisions about the
+development of the Perl language, the interpreter, and all other components,
+systems and processes that result in new releases of the language interpreter.
+
+The Steering Council does not have any authority over parts of the Perl
+ecosystem unrelated to developing and releasing the language itself.  Nor does
+it have the power to modify the Perl Rules of Governance.
+
+Decisions of the Steering Council will be made by majority vote of non-vacant
+seats on the council.
+
+### Election, Term, and Replacement
+
+Members of the Steering Council will be elected through ranked-choice voting
+by active members of the Core Team.
+
+A new Steering Council will be chosen by a Term Election within two weeks
+after each stable feature release.<a href="#note1"
+id="note1ref"><sup>1</sup></a> The council members will serve until the
+completion of the next Term Election unless they resign or are removed.
+
+Whenever there are vacancies on the Steering Council, the council will
+organize a Special Election within one week after the vacancy occurs.  If the
+entire Steering Council is ever vacant, a Term Election will be held instead.
+
+The election of the first Steering Council will be a Term Election.  Ricardo
+Signes will be the Vote Administrator for the initial Term Election unless he
+is a candidate, in which case he will select a non-candidate administrator to
+replace him.
+
+### Resignation, Absence, and Removal
+
+Steering Council members may resign their position at any time.
+
+If a Steering Council member is deceased, or drops out of touch and cannot be
+contacted for a month or longer, then the rest of the council may vote to
+declare their seat vacant.  If an absent member returns after such a
+declaration is made, they are not reinstated automatically, but may run in the
+Special Election to fill the vacancy.
+
+Otherwise, Steering Council members may only be removed before the end of
+their term by a No Confidence Vote by the Core Team.
+
+## Voting
+
+### Vote Administration
+
+Every vote requires a Vote Administrator who manages communication, collection
+of secret ballots, and all other necessary activities to complete the voting
+process.
+
+Unless otherwise specified, the Steering Council selects the Vote
+Administrator.
+
+A Vote Administrator must not be a member of the Steering Council nor a
+candidate or subject of the vote.  A Vote Administrator may be a member of the
+Core Team and, if so, may cast a vote while also serving as administrator.  If
+the Vote Administrator becomes a candidate during an election vote, they will
+appoint a non-candidate replacement.
+
+If the entire Steering Council is vacant or is the subject of a No Confidence
+Vote, then the Core Team will select a Vote Administrator by consensus.  If
+consensus cannot be reached within one week, the President of The Perl
+Foundation will select a Vote Administrator.
```

Comment:
Is TPF ok with adding that to their president's responsibilities? Does this mean the president can never also be a member of the Steering Council or Core Team?

Reply:
Again, cribbed from Python.  I think as written, nothing prohibits the President of TPF from the Core Team or Steering Council.  There are a series of clearly delineated fallbacks.  Even if the President is on the Steering Council and the whole Council is subject to No Confidence, if the Core Team can't agree by consensus, the President picks the Administrator because that rule overrrides.

Even what 'consensus' means is fuzzy, but I can't think of anything that can effectively function deterministically to pick someone to run a vote without getting into a complicated line of succession.  (Farcically, 'Larry or his heirs by absolute primogeniture?')

But if we're at a point of mass resignation or no-confidence and the TPF no longer exists and there's no President or they don't want to exercise the role, then things are so bad that I'm not really worried about what happens in this unlikely case.  Maybe there's another constitutional convention to reboot again.  Or maybe the Core Team bends the rules to find a way out and everyone goes with it.

Reply:
I'm with you on most of that; however it feels impolite to write an existing role in to this without getting the consent of the current holder.

Reply:
I was also concerned about this, but realistically it doesn't matter who picks the Vote Administrator as long as someone does.  (The Vote Administrator's role is relatively limited, so risk is low.  If they're going to rig an election, there are bigger issues.)

Reply:
Stuart has acknowledged and assented to this possibility.

Reply:
> Stuart has acknowledged and assented to this possibility.

Super, thanks.